Journal/Journal/EntryTableViewCell.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  UITableViewCell.swift
+//  Journal
+//
+//  Created by Dojo on 8/5/20.
+//  Copyright © 2020 Dojo. All rights reserved.
+//
+
+import UIKit
+
+class EntryTableViewCell: UITableViewCell {
+    
+    static var reuseIdentifier = "EntryCell"
+    
+    @IBOutlet weak var entryTitleLabel: UILabel!
+    @IBOutlet weak var descriptionLabel: UILabel!
+    @IBOutlet weak var timeLabel: UILabel!
+    
+    var entry: Entry? {
+        didSet {
+            updateViews()
+        }
+    }
+    
+    var dateFormatter: DateFormatter = {
+        let newDate = DateFormatter()
+        newDate.calendar = .current
+        newDate.dateFormat = "MM-dd-yyyy h:mm a"
+        return newDate
+    }()
```

Comment:
I know this is how Lambda had you do it, but just letting you know that you could just create a static date formatter and use that one formatter for all of your cells instead of having each cell create it's own date formatter:

``` swift
extension Date {
    var shortString: String {
        return DateFormatter.shortFormatter.string(from: self)
    }
}

extension DateFormatter {
    static let shortFormatter: DateFormatter = {
        let dateFormatter = DateFormatter()
        dateFormatter.dateStyle = .short
        dateFormatter.timeStyle = .short
        return dateFormatter
    }()
}
```

So then you could use it like so:

``` swift
timestampLabel.text = entry.timestamp.shortString
```